Sprint-1/1-key-exercises/1-count.js

```diff
@@ -4,3 +4,5 @@ count = count + 1;
 
```

Comment:
You can also use += operator in this  situation as well 
fyi 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Addition_assignment